client/components/app/MediaPlayerContainer.vue

```diff
@@ -11,7 +11,7 @@
           </nuxt-link>
           <widgets-explicit-indicator v-if="isExplicit" />
         </div>
-        <div class="text-gray-400 flex items-center w-1/2 sm:w-4/5 lg:w-2/5">
+        <div class="text-gray-400 flex items-center w-1/2 sm:w-4/5 lg:w-2/5 truncate">
           <span class="material-symbols text-sm">person</span>
           <div v-if="podcastAuthor" class="pl-1 sm:pl-1.5 text-xs sm:text-base">{{ podcastAuthor }}</div>
           <div v-else-if="authors.length" class="pl-1 sm:pl-1.5 text-xs sm:text-base truncate">
```

Comment:
truncate is already being added here and works for me in brave browser.
Were you running the latest server version or are you using a different browser?

Fixed in v2.11.0 as reported here: https://github.com/advplyr/audiobookshelf/issues/3038

Reply:
ah, maybe I hadn't updated yet. I'll double check that. But I had branched off of master and didn't see truncate in there.

Reply:
I still see the problem in firefox, and I've updated to the latest version: 
![image](https://github.com/user-attachments/assets/e8776e48-cca0-47c4-9fdd-30e8f72f37f9)

Reply:
I think it's worth noting that the fix you linked seems to have been for multiple authors, but this podcast only has one very long author. So truncating the author container fixes all possible author types being too long.

Reply:
resolving this since it's not really relevant since my last push.